CONTRIBUTING.md

```diff
@@ -74,9 +74,11 @@ Run a single test
 `$ rake test TEST=path/to/test.rb TESTOPTS="--name=test_something"`
 
 Run tests against different Rails versions by setting the RAILS_VERSION variable
-and bundling gems.
+and bundling gems.  (save this script somewhere executable and run from top of AMS repository)
```

Comment:
for example, this script is now out of date since we don't run on 4.0 anymore.  wwtd just using the travis file as a source of truth.  you could, of course do something like `ruby -ryaml -e 'puts YAML.load_file("./.travis.yml")["env"]["matrix"]' | xargs` instead to make the list